build-system/compile/bundles.config.extensions.json

```diff
@@ -434,6 +434,7 @@
         "amp-story-draggable-drawer-header",
         "amp-story-hint",
         "amp-story-info-dialog",
+        "amp-story-page-attachment",
```

Comment:
Working on separating these in #33042.

Reply:
The styles for`.i-amphtml-story-page-attachment-expand` aren't being applied.
Code pointer to where the element is created:
https://github.com/ampproject/amphtml/blob/master/extensions/amp-story/1.0/amp-story-page-attachment.js#L274-L280

 I think it's worth separating the style sheets in this PR to reduce side effects.

Reply:
I think this issue is unrelated to refactoring, because either way we'd need to create a new file and a shadow DOM and import the CSS like this. I'm investigating this issue now

Reply:
I've tried separating the style sheets but that style class continues to be ignored. This style is only rendered if it's imported in `amp-story.css`, which shouldn't be the case bc we're importing it directly in `amp-story-page.js`

Reply:
Ok, separated the stylesheets. Now, the styles for the shadow DOM element are imported directly in the JS, while the styles for everything else (the `.i-amphtml-story-page-attachment-expand` animation) are being imported in the `amp-story.css`